src/test/java/org/mockitousage/constructor/CreatingMocksWithConstructorTest.java

```diff
@@ -52,6 +55,12 @@ public void can_spy_abstract_classes() {
     }
 
     @Test
+    public void can_spy_abstract_classes_with_constructor_args() {
```

Comment:
Thank you very much for adding test coverage! I request more :)

As pointed out in earlier code review comments, we need coverage for ambiguous, overloaded constructor use cases. Perhaps they need better exception messages and intuitive behavior?